docs/data/core-edit-post.md

```diff
@@ -0,0 +1,271 @@
+# **core/edit-post**: The Editor's UI Data
+
+## Selectors 
+
+### getEditorMode
+
+Returns the current editing mode.
+
+*Parameters*
+
+ * state: Global application state.
+
+
+### isEditorSidebarOpened
+
+Returns true if the editor sidebar is opened.
+
+*Parameters*
+
+ * state: Global application state
+
+*Returns*
+
+Whether the editor sidebar is opened.
+
+### isPluginSidebarOpened
+
+Returns true if the plugin sidebar is opened.
+
+*Parameters*
+
+ * state: Global application state
+
+*Returns*
+
+Whether the plugin sidebar is opened.
```

Comment:
A lot of these have explanations that should be the return value and return values that better explain the function. Maybe it's worth switching them?